### PR TITLE
feat(DW): add random

### PR DESCRIPTION
**Pull request by pddpd**
- add random
- solve some test issues
- remove containsNonWorlds

### CODE REVIEW

File: packages/data-wizard/package.json [hunk 1]
```diff
@@ -50,16 +50,19 @@
...
     },
     {
       "path": "dist/index.min.js",
-      "limit": "30 Kb"
+      "limit": "200 Kb"
     }
   ],
   "dependencies": {
-    "@antv/ckb": "^2.0.0-alpha.1"
+    "@antv/ckb": "^2.0.0-alpha.1",
```
Comment (neoddish) [inline packages/data-wizard/package.json:62]: wait, why ckb...

Comment (pddpd) [reply packages/data-wizard/package.json:62]: We `import type { LevelOfMeasurement } from '@antv/ckb'` at analyzer. 🤔 but, why it's deleted at diff and added again.

Comment (neoddish) [reply packages/data-wizard/package.json:62]: ，

**Review by neoddish — APPROVED** [review]
LGTM

**Pull request merged**